### PR TITLE
fix: add documents to document plugin dataset

### PR DESCRIPTION
**Pull request by hguerni**

### CODE REVIEW

File: recipes/plugins_example_document.ipynb [hunk 1]
```diff
@@ -122,6 +122,7 @@
    ]
```
Comment (pmKili) [inline recipes/plugins_example_document.ipynb:122]: <pre class="ql-syntax" spellcheck="false">"""Plugin file"""


from kili.plugins import PluginCore
from kili.types import Label



class PluginHandler(PluginCore):
    """
    Custom plugin instance
    """


    @staticmethod
    def check_rules_on_label(label: Label):
        """
        Method for business logic
        """


        text_issues_array = []
        mid_issues_array = []


        for bbox in label['jsonResponse']['JOB_0']['annotations']:


            # Rule 1 - Check IBAN starts by FR
            if bbox['categories'][0]['name'] == "IBAN":
                iban = bbox['children']['TRANSCRIPTION_JOB']['text']


                if iban[0:2] != "FR":
                    text_issues_array.append("IBAN number should start by FR")
                    mid_issues_array.append(bbox['mid'])


            # Rule 2 - Check if Currency is in list of fields
            if bbox['categories'][0]['name'] == "CURRENCY":
                currency = bbox['children']['TRANSCRIPTION_JOB_2']['text']


                if currency not in ["DOLLAR", "EURO"]:
                    text_issues_array.append(
                        "Authorized currencies are only Euro and Dollar"
                        )
                    mid_issues_array.append(bbox['mid'])


        return text_issues_array, mid_issues_array


    def on_submit(self, label: Label, asset_id: str) -&gt; None:
        """
        Dedicated handler for Submit action
        """
        self.logger.info("On submit called")


        text_issues_array, mid_issues_array = self.check_rules_on_label(label)


        project_id = self.project_id


        n_issues = len(text_issues_array)


        for i in range(n_issues):


            self.kili.append_to_issues(
                label_id=label['id'],
                project_id=project_id,
                text=text_issues_array[i],
                object_mid=mid_issues_array[i]
            )


        self.kili.add_to_review(asset_ids=[asset_id])


        n_annotations = len(label['jsonResponse']['JOB_0']['annotations'])


        accuracy = (1 - n_issues / n_annotations) * 100


        self.kili.update_properties_in_assets(asset_ids=[asset_id], json_metadatas=[
            f"{{'accuracy': {accuracy}}}"
            ])


</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/989/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (pmKili) [reply recipes/plugins_example_document.ipynb:122]: <p>Can you use the following snippet instead?</p>

File: recipes/plugins_example_document.ipynb [hunk 2]
```diff
@@ -194,12 +191,16 @@
...
+    "            100 - len(issues_array) / len(label[\"jsonResponse\"][\"JOB_0\"][\"annotations\"]) * 100\n",
+    "        )\n",
     "\n",
-    "        print(\"Accuracy score computed\")\n",
+    "        print(accuracy)\n",
+    "        self.kili.update_properties_in_assets(\n",
+    "            asset_ids=[asset_id], json_metadatas=[\"{'accuracy': accuracy}\"]\n",
+    "        )\n",
     "\n",
-    "        kili.update_properties_in_assets(asset_ids=[asset_id], json_metadatas=[{'accuracy': accuracy}])"
```
Comment (p-desaintchamas) [inline recipes/plugins_example_document.ipynb:202]: where do we use all the billing documents ?

Comment (p-desaintchamas) [reply recipes/plugins_example_document.ipynb:202]: it's better to avoid image files in a repo, it is heavy & we can simply use URLs

Comment (hguerni) [reply recipes/plugins_example_document.ipynb:202]: in the notebook !

Comment (hguerni) [reply recipes/plugins_example_document.ipynb:202]: I should put those in a bucket then ? The problem is that the notebook is a form of example to upload asset from your computer as well

File: recipes/plugins_example_document.ipynb [hunk 3]
```diff
@@ -93,8 +93,24 @@
     "\n",
```
Comment (gheorghetutunaru) [inline recipes/plugins_example_document.ipynb:93]: <pre class="ql-syntax" spellcheck="false">Line #37.    kili.update_properties_in_assets(
</pre><p>What is this for ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/989/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: recipes/plugins_example_document.ipynb [hunk 4]
```diff
@@ -93,8 +93,24 @@
...
     "path = \"./datasets/document_plugin_data/\"\n",
```
Comment (gheorghetutunaru) [inline recipes/plugins_example_document.ipynb:94]: You don't use this

File: recipes/plugins_example_document.ipynb (hunk 3 above)
Comment (hguerni) [reply recipes/plugins_example_document.ipynb:93]: It was to add metadatatype but I deleted a part of it, fixed !

**Review by florianlega — APPROVED** [review]

**Review by gheorghetutunaru — APPROVED** [review]

**Pull request merged**